app/config.go

```diff
@@ -782,7 +827,8 @@ func ResetWorldState(cmd *cobra.Command, args []string) {
 	if cmd.Flag("datadir").DefValue == cmd.Flag("datadir").Value.String() {
 		home, err := os.UserHomeDir()
 		if err != nil {
-			panic("could not get home directory for data dir creation: " + err.Error())
+			fmt.Println("could not get home directory for data dir creation: " + err.Error())
+			os.Exit(1)
```

Comment:
```suggestion
                      log.Fatalf("could not get home directory for data dir creation: %s", err.Error())
```
stdlib package `log` offers `log.Fatalf` which logs the error with a timestamp & calls onto os.Exit(1)

the `log` package needs to also be added on this file for this suggestion to work